src/components/books/books.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import AddBook from './CreateNewBook';
+
+function Books() {
+  return (
+    <>
+      <AddBook />
+      <div className="d-flex justify-content-center mt-5">
+        <table>
+          <tr>
+            <th>
+              <b>
+                &quot;Exquisite Mariposa: A Novel&quot;
+              </b>
+            </th>
+            <th>by</th>
+            <th>&quot; Fiona Alison Duncan&quot;</th>
+            <th><button type="button" className="btn btn-outline-dark btn-sm">Remove</button></th>
+          </tr>
```

Comment:
- [ ] As it is a requirement for this project and a best practice for `react`, i.e. `making the building blocks of your app set as re-usable components`, it will be nice to create another component called for example `Book`, that;
- Takes the book data as a prop and renders the `title` and the `author`,
- Has a button for removing books(the button wouldn't remove the books yet, this is just a placeholder for now
).
It is similar to  what you have done here, but it will be nice for you to create a component and render it here, like you have done for the `<AddBook />` above. See screenshots below for more clarifications.

![components](https://user-images.githubusercontent.com/65852621/151108290-6672b7a9-1661-4e6c-b9b3-91dd85d92c90.png)

![componets2](https://user-images.githubusercontent.com/65852621/151108303-817968cd-fe5c-4883-8497-fe6fee1d96c2.png)

Reply:
I have fixed it!! Thanks, Julius, for the suggestion to follow good practices!